templates/master/00-master/gcp/files/etc-kubernetes-manifests-gcp-routes-controller.yaml

```diff
@@ -15,7 +15,7 @@ contents:
         command: ["gcp-routes-controller"]
         args:
         - "run"
-        - "--health-check-url=https://127.0.0.1:6443/readyz"
+        - "--health-check-url={{.Infra.Status.APIServerInternalURL}}/readyz"
```

Comment:
just to be sure, is this written once or dynamically updated? Afaik in theory APIServerInternalURL could change?

Reply:
I don't think anything supports this changing. AFAICT the "status" is generated in the installer.

Reply:
ok. then it's fine.